src/userinput.c

```diff
@@ -42,6 +42,8 @@ static char *uri_escape(const char *string)
 		if (allocated_len + 4 >= real_len) {
 			allocated_len += 16;
 			escaped = realloc(escaped, allocated_len);
+			// bail out if realloc fails
+			if (escaped == NULL) return NULL;
```

Comment:
The [Linux kernel coding style](https://www.kernel.org/doc/html/v4.10/process/coding-style.html#placing-braces-and-spaces) seems to suggest `if` and `return` should be split into two lines. I agree that's unclear but that's the style all examples use.

Reply:
done.